botorch/utils/sampling.py

```diff
@@ -154,22 +158,30 @@ def draw_sobol_samples(
             to lower and upper bounds, respectively.
         n: The number of (q-batch) samples.
         q: The size of each q-batch.
+        batch_shape: The batch shape of the samples. If given, returns samples
+            of shape `n x *batch_shape x q x d`, where each batch is an
+            `n x q x d`-dim tensor of qMC samples.
         seed: The seed used for initializing Owen scrambling. If None (default),
             use a random seed.
 
     Returns:
-        A `n x q x d`-dim tensor of qMC samples from the box defined by bounds.
+        A `n [x *batch_shape] x q x d`-dim tensor of qMC samples from the box
```

Comment:
we usually just use the following notation: `n x batch_shape x q x d`, understanding that `batch_shape` is a (possibly 0-element) tuple.